Cartfile.resolved

```diff
@@ -1,6 +1,6 @@
 binary "https://mapbox-gl-native-ios.s3.amazonaws.com/public/internal/Mapbox-iOS-SDK.json" "5.9.1000"
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" "2.3.0"
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" "14.1.5"
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" "14.1.6"
```

Comment:
Not sure if update to nav. native `14.1.6` should be part of this changelist.

Reply:
that's `Cartfile.resolved`. This PR does not bump any dependencies. Anyway, after rebasing to the latest commit such change should vanish.